grakn-test/test-integration/src/test/java/ai/grakn/test/migration/owl/TestSamplesImport.java

```diff
@@ -42,16 +42,16 @@
 import static org.junit.Assert.assertTrue;
 
 /**
- * Load and verify the ontology from the test sample resources. 
- * 
+ * Load and verify the schema from the test sample resources.
+ *
  * @author borislav
  *
  */
 public class TestSamplesImport extends TestOwlGraknBase {
 
     @Ignore //TODO: Failing due to tighter temporary restrictions
     @Test
-    public void testShoppingOntology()  {       
+    public void testShoppingSchema()  {
```

Comment:
It converts an OWL ontology into a Grakn Schema . . . Yes that's the excuse I am going with.